src/data/polynomial/laurent.lean

```diff
@@ -294,6 +294,15 @@ left_inverse_trunc_to_laurent.injective
   f.to_laurent = g.to_laurent ↔ f = g :=
 ⟨λ h, polynomial.to_laurent_injective h, congr_arg _⟩
 
+instance : module R[X] R[T;T⁻¹] :=
```

Comment:
This is safe, but the corresponding  `module S[X] R[T;T⁻¹]`

would not be.

Reply:
Great, thanks for confirming!  I was trying to extract the less controversial instances first, from the left-over Laurent stuff!